README.md

```diff
@@ -4,7 +4,6 @@ A small cli tool to generate TOTP (Time-based One-time Password) tokens.
 
 ## Usage
 
-```
-Usage of otpgen:
-  -s, --secretkey string   supply a valid TOTP secret key to generate a token from
+```bash
+otpgen -s|--secretkey SECRETKEY
```

Comment:
IMO, we should stick to the help output of flagp library. Why should we change that?

Reply:
You are absolutely right! I reset the changes.